developer_document.md

```diff
@@ -499,9 +492,21 @@ index 894be0134d..a9a57c0a9e 100644
  {
    m.doc() = "cpp model python binding";
 ```
+# 3.	Accuracy evaluation
+## 3.1 Evaluate llm model in neural speed by lm_eval
+We can use this python script for accuracy evaluation.
+```
+python scripts/cal_acc.py --model hf_model --tasks piqa --group_size 32 --compute_dtype int8 --weight_dtype int4
+```
+
+Here is lm-evaluate result for [Intel/neural-chat-7b-v3-1](https://huggingface.co/Intel/neural-chat-7b-v3-1) in weight_dtype=int4, compute_dtype=int8, group_size=-1 quant config.
+
+| lambada_openai | boolq | truthful_qa_mc1 | piqa | winogrande | openbookqa | mmlu | hellaswag | arc_easy | arc_challenge |
```

Comment:
update table of https://huggingface.co/Intel/neural-chat-7b-v3-3-int4-inc and let user know our acc script and inference has no problem.

Reply:
done